data/wanderer/wanderers start.txt

```diff
@@ -1906,6 +1917,7 @@ mission "Wanderers Hai Assistance 3"
 			`	"But people will die, on your side as well as the Wanderers," says Sayari. "They are willing to give you these worlds if you only give them time, perhaps no more than a few months or a year. Why lose your lives to claim what will be yours soon enough?"`
 			`	"Better to die asserting our strength than to live with our hands outstretched to beg," says another of the chiefs.`
 			`	You are unable to convince them to change their plans, and as soon as the conversation ends they insist that you leave their planet. "Our warriors in orbit are eager to test this human's strength," says one of the chiefs, laughing.`
```

Comment:
The supreme commander of the Hai military was already thoroughly impressed by Captain `<last>`'s strength in battle. The chiefs would fear and respect Captain `<last>`, as would the warriors in orbit.

Reply:
If you want to take a pass at this, please do so. Otherwise I'll work on it after I fix the mission coding bugs.

Reply:
A pass has been taken.